src/apprt/gtk-ng/class/application.zig

```diff
@@ -751,6 +755,40 @@ pub const Application = extern struct {
         );
     }
 
+    fn startupGtkActions(self: *Self) void {
```

Comment:
My branch is going to conflict with this probably... ill PR mine shortly.

Reply:
Yeah, but I've synced it with your branch so conflicts should be minimal.